apps/src/weblab/WebLab.js

```diff
@@ -615,4 +615,12 @@ WebLab.prototype.PageAction = makeEnum(
   logToCloud.PageAction.BrambleFilesystemResetFailed
 );
 
+WebLab.prototype.redux = function() {
```

Comment:
What is this for?

Reply:
CdoBramble needs to access our redux store, but isn't able to load the store directly because it is loaded in an iframe. so this method gives CdoBramble access to just the WebLab redux (CdoBramble reads from the store to generate Bramble config, and fires events like turning the inspector on/off)